website/activemembers/admin.py

```diff
@@ -25,14 +26,9 @@ class MemberGroupMembershipInlineFormSet(forms.BaseInlineFormSet):
     def __init__(self, *args, **kwargs):
         """Initialize and set queryset."""
         super().__init__(*args, **kwargs)
-        now = timezone.now()
-        self.queryset = self.queryset.select_related("member", "group")
-
-        # Show only memberships that are active now.
```

Comment:
Good catch! I think what happened is that when I copied the code from the older branch (which was a mess), I accidentally copied the old implementation for this class as well. I've fixed it now locally.

Unfortunately, I've also found another issue, because it seems like it's not possible to export only selected users from the separate Member export page. You can only export every member. I'll see if I can find a solution before pushing.